plugin/trino-hive/src/main/java/io/trino/plugin/hive/metastore/thrift/ThriftMetastoreUtil.java

```diff
@@ -740,25 +742,28 @@ public static Set<HivePrivilegeInfo> parsePrivilege(PrivilegeGrantInfo userGrant
         };
     }
 
-    public static HiveBasicStatistics getHiveBasicStatistics(Map<String, String> parameters)
+    public static HiveBasicStatistics getHiveBasicStatistics(Map<String, String> parameters, Boolean useSparkFallback)
```

Comment:
Why do we have to add an extra parameter to this method when in most of the cases it is `false` ?
Let's have this method stay as it is and call internally

````
return getHiveBasicStatistics(parameters, false)
```

The newly introduced method would be used only for `ThriftHiveMetastore` and `GlueHiveMetastore`